<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addIssue</class>
 <widget class="QDialog" name="addIssue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Month:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>January</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>February</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>March</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>April</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>May</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>June</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>July</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>August</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>September</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>October</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>November</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>December</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="spinBox_2">
     <property name="minimum">
      <number>1900</number>
     </property>
     <property name="maximum">
      <number>2100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spinBox">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Issue Number:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Year:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Journal Name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addIssue</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addIssue</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
